src/pytest/pytestTestRunner.ts

```diff
@@ -174,6 +132,27 @@ pytest.main(sys.argv[1:], plugins=[PythonTestExplorerDiscoveryOutputPlugin()])`;
         return states;
     }
 
+    private async loadEnvironmentVariables(config: IWorkspaceConfiguration): Promise<IEnvironmentVariables> {
+        const environment = await EnvironmentVariablesLoader.load(config.envFile(), process.env, this.logger);
+
+        const updatedPythonPath = [
+            config.getCwd(),
+            environment.PYTHONPATH,
+            DISCOVERY_OUTPUT_PLUGIN_INFO.PACKAGE_PATH
+        ].filter(item => item).join(path.delimiter);
```

Comment:
It would be [`os.pathsep`](https://docs.python.org/3/library/os.html#os.pathsep) in Python, but the equivalent is [`path.delimiter`](https://nodejs.org/api/path.html#path_path_delimiter) in node.